tests/foreman/ui/test_dashboard.py

```diff
@@ -186,23 +186,24 @@ def test_positive_task_status(session):
 
 
 @pytest.mark.upgrade
+@pytest.mark.no_containers
 @pytest.mark.run_in_one_thread
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
+@pytest.mark.rhel_ver_match('8')
```

Comment:
Non blocking: 
If there is a plan to parametrize this test for the other rhel_clients then it's good to use this marker, else you can switch to use the rhel8_contenthost fixture for the client.

Reply:
Good to keep this marker in case we want to switch to test rhel9. It's easier to modify the test for testing other rhel versions.